### DISCUSSION TITLE
TiVo Mini Network Remote for Windows

### DISCUSSION THREAD

**Discussion by tom76017** [GerryDazoo/Slinger#84] [Show and tell]
Here's a pretty slink solution for Windows VLC users that want a floating remote.  It talks direct to your TiVo.

https://github.com/wmcbrine/tivoremote

Setup is similar to the Peanut on the iPhone.  See https://github.com/GerryDazoo/Slinger/discussions/30

I don't have a Mac, but it should work there also.

**Comment by rblancarte** [comment]
Hmm, this could be interesting - pulling video in this way (Slinger) and then using a network remote to change channels on the TiVo directly.  🤔
